.github/workflows/build-images.yaml

```diff
@@ -1,5 +1,7 @@
 name: "Build+Push Images"
 on: # build on main branch OR when a PR is labeled with `CICD:build-images`
+  # Allow us to run arbitrary workflow without a PR
```

Comment:
i think the comment is a bit misleading. What this allows you actually is to run this *specific* workflow manually instead of allowing arbitrary workflows to be run